src/lib/pdfx-theme-context.tsx

```
@@ -0,0 +1,34 @@
+    <PdfxThemeContext.Provider value={resolvedTheme}>
+      {children}
+    </PdfxThemeContext.Provider>
+  );
+}
+
+export function usePdfxTheme(): PdfxTheme {
+  return useContext(PdfxThemeContext);
+}
+
+export function useSafeMemo<T>(factory: () => T, deps: DependencyList): T {
+  return useMemo(factory, deps);
```

ESLint

[error] Error: Expected the first argument to be an inline function expression

Expected the first argument to be an inline function expression.

/home/jailuser/git/src/lib/pdfx-theme-context.tsx:33:18
  31 |
  32 | export function useSafeMemo<T>(factory: () => T, deps: DependencyList): T {
> 33 |   return useMemo(factory, deps);
     |                  ^^^^^^^ Expected the first argument to be an inline function expression
  34 | }
  35 |

(react-hooks/use-memo)